app/controllers/admin_controller.rb

```diff
@@ -16,7 +16,7 @@ def index
       search_term = "%#{params[:search]}%"
       @users = @users.where('full_name LIKE ? OR email LIKE ?', search_term, search_term)
     end
-    @events = Event.where('"eventTime" > ?', Time.now)
+    @events = Event.all # where('"eventTime" > ?', Time.now)
```

Comment:
When you are rendering past and upcoming event, are we showing every single past event? That should be fine, But this definitely needs to go into an accordion so its not a chore to access everything below if there is alot of events.